benchmarks/src/main/java/okhttp3/benchmarks/OkHttp.java

```diff
@@ -40,15 +39,15 @@
         .build();
 
     if (benchmark.tls) {
-      SSLContext sslContext = SslContextBuilder.localhost();
-      SSLSocketFactory socketFactory = sslContext.getSocketFactory();
+      SslContextBuilder sslContextBuilder = SslContextBuilder.localhost();
```

Comment:
Ahh, but this makes sense. . . . should we have a different name than `SslContextBuilder` ? Throughout OkHttp `Builder` almost always means a mutable thing that builds an immutable thing. This feels awkward here.

(not necessarily new with your change, I’m just reflecting on the API!)